src/TasksModule/Components/TasksList/TasksList.tsx

```diff
@@ -1,7 +1,161 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+import { baseUrl } from "../../../Constants/Components/Urls.js";
+import axios from "axios";
+import { toast } from "react-toastify";
+import { useNavigate } from "react-router-dom";
+import NoData from "../../../noData/noData.js";
 
 export default function TasksList() {
+  const [taskList, setTaskList] = useState([]);
+  const [searchByTitle, setSearchByTitle] = useState("");
+  const [statusSearch, setStatusSearch] = useState([]);
+  const [pagesArray, setPagesArray] = useState([]);
+
+  const token = localStorage.getItem("userToken");
+  const getTitleValue = (e: string) => {
```

Comment:
This wasn't what I meant, we should be passing the title not the event itself.
Currently, you'll see warnnings in the application because we are trying to access `e.target.value` inside the function and we are saying that `e` is a string, not an event.
![image](https://github.com/SalmaAboArab/Project-Management-System/assets/64327685/0bc940fb-722f-4094-a760-57733b8c1920)